src/components/diagrams/diagram-pane.js

```diff
@@ -348,9 +354,24 @@ export function DiagramPane({
     const viewsRef = useRef();
     viewsRef.current = views;
 
-    const displayedDiagrams = views.filter((view) =>
-        [ViewState.OPENED, ViewState.PINNED].includes(view.state)
-    );
+    const sortByAlign = (a, b) => {
+        if (!a || !b) {
+            return 0;
```

Comment:
ne respecte pas les invariants de tri a < b  et b = c => a<c

Reply:
```
const signLikeOf = (align) => {
  if (align === 'left') return -1;
  if (align === 'right') return 1;
  return 0;
}

const displayedDiagrams = views
    .filter((view) =>
        [ViewState.OPENED, ViewState.PINNED].includes(view.state)
    )
    .sort((a, b) => signLikeOf(a.align) - signLikeOf(b.align));

```
?

Reply:
Yes, the old function was bad.
It is fixed in this other PR : https://github.com/gridsuite/gridstudy-app/pull/1055

Now waiting for the other PR to be merged, I am setting this one WIP.

Reply:
Ok, the sort and align aspect of this PR was merged in main and now this PR is simplified. Thank you for your patience.